pkg/accounts/service/tokenGenerator.test.ts

```diff
@@ -9,8 +10,8 @@ describe('TokenGenerator', () => {
   it('verify JWT Token', async () => {
     const token = await generator.generate(
       '',
-      new Date(),
-      new Date('2099/12/31 12:59:59'),
+      calculateDiffFromEpoch(new Date()),
+      calculateDiffFromEpoch(new Date('2099/12/31 12:59:59')),
```

Comment:
```suggestion
      calculateDiffFromEpoch(new Date('2099-12-31T12:59:59Z')),
```
この `/` で区切る書き方は処理系によって動かない書き方なので, ISO 8601 ライクの書き方にしたほうがいいと思います. この後の 26, 27 行目なども同様です.